### ISSUE TITLE
Plugins are being retired

### ISSUE THREAD

**Issue by karelkrivanek** [trakt/Plex-Trakt-Scrobbler#509]
Sadly, plex have announced, that they retiring plugins in near future - https://www.plex.tv/blog/subtitles-and-sunsets-big-improvements-little-housekeeping/ (about half way down, the first skull)

**Comment by Generator** [comment]
> But don’t panic—while the Plugin Directory will soon be gone, you can still manually install plugins for the foreseeable future.

You can still install plugins manually or with Unsupported AppStore (WebTools plugin)

**Comment by Generator** [comment]
Don't thing that using another third party is the best solution, i have Tautulli, but most of the users don't. 
Moving to standalone connect to Plex via API is the best solution (basicly is how third party software connects to Plex, like Tautulli, Ombi ...)

**Comment by vincegre** [comment]
> Moving to standalone connect to Plex via API is the best solution (basicly is how third party software connects to Plex, like Tautulli, Ombi ...)

Yep but would be nice to get that plugin ported to API system before plugins feature disappear completely of Plex now that it's clearly a dead end feature (plugins in Plex !).
Hope that Traktv moves forward on time :/
